test-scripts/resourceOwnerPasswordGrant.py

```diff
@@ -53,16 +52,15 @@ class OAuthConfig(Config):
     ONS_TOKEN_ENDPOINT = "/api/v1/tokens/"
 
 
-
-client_id='onc@onc.gov'
+client_id = 'onc@onc.gov'
 client_secret = 'password'
 username = 'testuser@email.com'
 password = 'password'
 
-print " *** Variables setup ***"
+print(" *** Variables setup ***")
```

Comment:
Can we make this a log? I only mention this because there was an update to make it python3 specific. :-)